### PR TITLE
Add (empty) chat information pages (popups)

### LINKED ISSUE #347: Popup for Groups
In order to place information such as group descriptions or member lists, a visual element accessible from the group chat is needed.

### PR DESCRIPTION
**Pull request by rsommerfeld**
resolves #347

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -28,8 +28,3 @@ jobs:
         timeout-minutes: 15
         env:
            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      - uses: actions/upload-artifact@master
-        if: ${{ always() }}
-        with:
-          name: Test Artifacts (${{ matrix.smalltalk }} (${{ matrix.os }}))
-          path: /home/runner/.smalltalkCI/_builds/telegramTestResults
```
Comment (rsommerfeld) [inline .github/workflows/main.yml:35]: I think we should undo this change 😛

Comment (jb3rndt) [reply .github/workflows/main.yml]: Yup xD

File: packages/TelegramClient-UI.package/TCUChatInfoPage.class/class/newBounds..st [hunk 2]
```diff
@@ -0,0 +1,6 @@
+instance creation
+newBounds: bounds
+
+	^ (super newBounds: bounds)
+		addInfoBar;
+		yourself.
```
Comment (Storyxx) [inline packages/TelegramClient-UI.package/TCUChatInfoPage.class/class/newBounds..st]: No dots after return

File: packages/TelegramClient-UI.package/TCUChatWindow.class/instance/createTitleBar.st [hunk 3]
```diff
@@ -6,9 +6,11 @@ createTitleBar
...
+		topLeft: self topLeft;
 		layoutPolicy: TableLayout new;
 		listDirection: #leftToRight;
 		layoutInset: 2;
 		vResizing: #shrinkWrap;
-		cellInset: 12).
\ No newline at end of file
+		cellInset: 12).
+	
+	self titleBar on: #mouseUp send: #showInfoPage to: self.
```
Comment (Storyxx) [inline packages/TelegramClient-UI.package/TCUChatWindow.class/instance/createTitleBar.st:16]: Is there a reason for using "mouseUp" here? From a little bit of testig, Squeak does not do what would be expected from using this signal. (If you click the group title, drag away and then release the mouse, you still activate the info-page)

File: packages/TelegramClient-UI.package/TCUChatWindow.class/instance/addHiddenInfoPage.st [hunk 4]
```diff
@@ -0,0 +1,8 @@
+drawing
+addHiddenInfoPage
```
Comment (permler) [inline packages/TelegramClient-UI.package/TCUChatWindow.class/instance/addHiddenInfoPage.st:2]: I think the name addInfoPage would be fine. Is it really necessary to specify that it is hidden?

Comment (rsommerfeld) [reply packages/TelegramClient-UI.package/TCUChatWindow.class/instance/addHiddenInfoPage.st:2]: The reason was, that there were quite a few other add... methods and all of them visually changed the window. In that context, one would expect to also see an info page after calling "addInfoPage" so we wanted to make sure hiding is in the title

File: packages/TelegramClient-UI.package/TCUChatWindow.class/instance/createTitleBar.st (hunk 3 above)
Comment (rsommerfeld) [reply packages/TelegramClient-UI.package/TCUChatWindow.class/instance/createTitleBar.st:16]: No, not really any good reason. I think we saw mouseUp being used at some other place of the telegram UI but you're right, mouseDown is better. Changed it now

**Review by tom-richter — APPROVED** [review]
